### DISCUSSION TITLE
Disable fulfillment

### DISCUSSION THREAD

**Discussion by virajsoni06** [swellstores/community#468] [Help]
Hi we have a site with digital products and don't want to enable fulfillment, how can it be disabled? 
Also is it a new feature added? Our order flow used to work perfectly fine before. Something seems to have changed on Swell's end

**Comment by logeshswell** [comment]
Can you please share your sample product link for reference?
If your product is declared as digital, then shipping (i.e fulfillment) will be disabled.

**Reply by virajsoni06** [reply]
Hey, nevermind this one. There were some product misconfigs, figured it out. Thanks
